### PR TITLE
Adding ability to unwatch an element

### PR DESCRIPTION
**Pull request by weotch**
By calling `balanceText(el, {watch: false})` this allows you to stop watching an element.  My use case for wanting this is in single page apps.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (redmunds) [inline yarn.lock]: This project does not yet support yarn, so this should be removed from this PR.

Comment (weotch) [reply yarn.lock]: Oops, didn't mean to commit that, thanks

**Comment by redmunds** [comment]
@weotch Thanks for the contribution. I might not be able to review this until next week.

**Comment by redmunds** [comment]
Thanks. FYI, I added a test file in PR #114.

**Pull request merged**